### DISCUSSION TITLE
User guide is ready to bring into the project

### DISCUSSION THREAD

**Discussion by podfeet** [bartificer/xkpasswd-js#89] [General]
I have completed the user guide for XKPasswd, and it's in my repo at [github.com/podfeet/docsify-user-docs-xkpasswd](https://github.com/podfeet/docsify-user-docs-xkpasswd).

I'd like to have a discussion of the process for incorporating this work into the xkpasswd-js repo.  I don't want these documents just taken an incorporated, I'd like to participate in the decisions on how this is done.

**Comment by bbusschots** [comment]
I’ve forked your repo and done a test of running it on a subdomain and I think it looks good! https://test.docs.xkpasswd.net/

So, I think the next steps are to:

1. Transfer your repo into the Bartificer org
2. Decide on a subdomain and then set it up (more on that below)
3. Add appropriate verbiage and links to the password generator

In terms of domains, I can think of three sane candidates, and am open to alternative suggestions:

1. Docs.xkpasswd.net
2. Manual.xkpasswd.net
3. About.xkpasswd.net

I not actually that keen on using docs because to me that’s where the developer docs for the library should end up some day. Manual is probably the most technically correct, but it sounds a little dry to me. So, my current preference would be about, but I’m very open to being swayed on this.

**Reply by podfeet** [reply]
Step 0 - add podfeet to the Bartificer org
Step last - figure out where in the XKPasswd interface to point to the documentation.

On naming:

4. UserManual.xkpasswd.net
5. UserGuide.xkpasswd.net

They're longer but when you get to the user guide you'd think, "It does what it says on the tin."

**Reply by bbusschots** [reply]
You've changed my mind — my preferred option is now `userguide.xkpasswd.net`

**Reply by hepabolu** [reply]
I agree with userguide. 

We can create a menu item in xkpass that points to this site.

**Comment by bbusschots** [comment]
OK, we seem to have consensus on `userguide.xkpasswd.net` for the domain.

The next step is to move the repo into the org from where I can re-name it and plumb in the DNS.

@podfeet, you need to push the repo to the org by navigating to the repo, opening the settings for the repo and then scrolling down to the bit humorously labeled *'Danger Zone'* and clicking the `Transfer` button.

**Reply by podfeet** [reply]
I asked that step 0 be that I'm added to the org.

**Comment by podfeet** [comment]
I've transferred ownership. I'm very curious how this will work - can't picture it in my head where it fits into the structure of the code. Was hoping there'd be a discussion of the architecture...

**Reply by hepabolu** [reply]
It will be a separate project. I created an issue to add a link to the user guide in the top menu

**Comment by podfeet** [comment]
Ah - that makes more sense! the repo name is kind of dopey - I was trying a whole pile of static site generators and so I had to name them appropriately to tell them apart. It's not terrible but certainly not concise.

**Comment by hepabolu** [comment]
Since the user documentation is available under the Bartificer organization and up and running under a sensible domain name, can we close this discussion?

**Reply by bbusschots** [reply]
I think you can close
